src/one_dragon/base/operation/application/plugin_import_service.py

```
@@ -0,0 +1,494 @@
+                    return ImportResult(
+                        success=False,
+                        plugin_name=zip_path.stem,
+                        message="无法确定插件目录名"
+                    )
+
+                target_dir = self.plugins_dir / plugin_dir_name
+
+                # 检查目录是否已存在
+                if target_dir.exists():
+                    if overwrite:
+                        # 覆盖安装：先删除旧目录
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)